Newsletter for Feb 2020 (#830)
alan-turing-institute/the-turing-way: communications/newsletters/newsletter_11_Feb2020.md

One of the past attendees, [Jade Pickering](https://twitter.com/Jade_Pickering) will be returning as a helper along with the core contributors [Louise Bowler](https://www.turing.ac.uk/people/researchers/louise-bowler) and [Sarah Gibson](https://twitter.com/drsarahlgibson).
Those who are not located in London have also received travel and accommodation support from *The Turing Way* project.
We are looking forward to welcoming them and working with them in person!

#### Your comments are needed on the rubric for selection

> A call for application was opened in December 2019, inviting applications from the interested candidates sharing their ideas for contribution to *The Turing Way* project.

Author: ```suggestion
An open call for applications was opened in December 2019 for the book dash, inviting ideas from the interested candidates for their contributions to *The Turing Way* project.
```

Author: As a separate (independent paragraph), I had not clarified that we are still talking about the book dash.

They were also asked to justify how they will benefit from their participation at this event.
We adapted the Mozilla Open Leadership selection rubric for ranking and scoring all the applications.
This rubric is currently available in [pull request #804](https://github.com/alan-turing-institute/the-turing-way/pull/804).
We welcome your comments and suggestions on this rubric for the selection process in our future events.

### Translating *The Turing Way* into additional languages

Ang Li and Yini He have translated four initial chapters of *The Turing Way* in Chinese. 
These researchers from China are spending a year in The Alan Turing Institute in the [Whitaker Lab](https://twitter.com/Whitaker_Lab) and are working on different topics in the neurobiology field while contributing to *The Turing Way* project.

![Tweet from Turing Way project and Kirstie Whitaker celebrating Tony, Ang & Yini's progress translating the book](/communications/newsletters/images/Tweet-Translation-Feb2020.png)
*Read the [full thread on Twitter](https://twitter.com/turingway/status/1225719523497697280)*

> Tony Yang, who is conducting his PhD research at Imperial College London has set out to develop standardised guidelines and infrastructure for translation efforts in Chinese and other languages within the community.

Author: ```suggestion
Tony Yang, who is conducting his PhD research at Imperial College London. has set out to develop standardised guidelines and infrastructure for translation efforts in Chinese and other languages within the community.
```

Reviewer: Should be a comma not a period. I'll fix locally - thanks for point it out!

> He will be working with Turing Way team members during the book dash in integrating this workflow in the main project pipeline.

Author: ```suggestion
He will be working with _The Turing Way_ team members during the book dash in integrating this workflow in the main project pipeline.
```

A past book dash attendee, Camila Rangel-Smith and her colleague Reina Camacho Toro have expressed interest in beginning to translate a few existing chapters in Spanish.

> We are so grateful for their work so far and invite our Chinese- and Spanish-speaking members to join this effort. Connect with them [on this GitHub issue](https://github.com/alan-turing-institute/the-turing-way/issues/767).

Author: ```suggestion
We are so grateful for their work so far and invite our Chinese- and Spanish-speaking members to join this effort. Join this discussion [on this GitHub issue](https://github.com/alan-turing-institute/the-turing-way/issues/767).
```

Reviewer: Fixed locally

## Relevant Resources

### _The Turing Way_ on twitter

> We are finally on Twitter and are very happy to connect with many of you there.

Author: ```suggestion
We are finally on Twitter and have been thoroughly enjoying reading your tweets about us (previously tagged with [#TuringWay](https://twitter.com/hashtag/TuringWay?f=live)).
```

> Within hours of its launch 2 weeks ago we gained over 100 followers and delighted to read your tweets about us!

Author: ```suggestion
Within hours of its launch 2 weeks ago, we gained over 100 followers and have managed to connect with 400 followers at the time of writing. 
```

> Connect with us for updates on the project and learn what our members are up to.

Author: ```suggestion
Please follow the account [turingway](https://twitter.com/turingway) for updates on the project and learn what our members are up to. 
```

> Follow us here: [https://twitter.com/turingway](https://twitter.com/turingway).

Author: ```suggestion

```

See [the workshop page](https://alan-turing-institute.github.io/2020-02-10-turing/) for details.

### Impressions from Twitter

![Screenshots of tweets linked below from ADR UK, Louise Corti, Fiona Grimm and Ross Markello](communications/newsletters/images/Tweet-combo-Feb2020.jpg)

*Tweets by [ADR UK](https://twitter.com/adr_uk), [Louise Corti](https://twitter.com/LouiseCorti), [Fiona Grimm](https://twitter.com/fiona_grimm) and [Ross Markello](https://twitter.com/rossdavism).*

## Connect with us!

We love hearing about how you're using _The Turing Way_.
Stay in touch through one of the many different pathways below!

- [About the project](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)
- [_The Turing Way_ book](https://the-turing-way.netlify.com)
- [GitHub repository](https://github.com/alan-turing-institute/the-turing-way)
- [Gitter chat room](https://gitter.im/alan-turing-institute/the-turing-way)
- [YouTube Videos](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)

> - Twitter: [@turingway](https://twitter.com/turingway) or [#TuringWay](https://twitter.com/hashtag/TuringWay?f=live)

Author: ```suggestion
- [Twitter account](https://twitter.com/turingway)
  - Hashtag [#TuringWay](https://twitter.com/hashtag/TuringWay?f=live)
```
